core/src/main/kotlin/com/malinskiy/marathon/execution/strategy/StrictMode.kt

```diff
@@ -0,0 +1,7 @@
+package com.malinskiy.marathon.execution.strategy
+
+enum class StrictMode {
```

Comment:
In a sense this is not a strict mode anymore, it's a way to treat failures in the build. Not sure if there is a better naming

Reply:
I think the better could be `execution_strategy`

Reply:
Yeah, this sound better